pkg/python/uv.go

```diff
@@ -41,6 +41,7 @@ const (
 	pythonVersionForIngestr = "3.11"
 	ingestrVersion          = "0.14.101"
 	sqlfluffVersion         = "3.4.1"
+	adbcDriverManagerVersion = "1.2.0"
```

Comment:
why are you installing this much older version of the driver manager? this should probably be 1.9.0 not 1.2.0